### PR TITLE
Include license and DOI in the preprint PDF

### PR DESCRIPTION
**Pull request by leouieda**
We have a DOI for the preprint and it's licensed CC-BY (as per the
EarthArXiv terms). Add this information to the preprint, make the
margins a little smaller (I know...) and include the full citation in
the footer.

### CODE REVIEW

**Comment by leouieda** [comment]
@santisoler what do you think? We'd need to update the preprint on EarthArXiv.

File: manuscript/preprint.tex [hunk 1]
```diff
@@ -20,41 +20,56 @@
...
-% Configure column separation and page geometry
-%\usepackage[hmargin=2cm, vmargin=3cm]{geometry}
-%\setlength{\columnsep}{2\columnsep}
-
 % Import variables
 \input{variables.tex}
 
+% Configure column separation and page geometry
+\usepackage[hmargin=2cm,vmargin=2.9cm]{geometry}
+
```
Comment (santisoler) [inline manuscript/preprint.tex:28]: 🤣

File: manuscript/preprint.tex [hunk 2]
```diff
@@ -20,41 +20,56 @@
...
-        \Uieda$^{3}$\orcidlink{\UiedaORCID}
+      \Soler$^{1,2}$ \orcidlink{\SolerORCID} and
+        \Uieda$^{3}$ \orcidlink{\UiedaORCID}
     }
     \\[0.4cm]
     {\small $^{1}$\CONICET (\SolerMail)} \\
     {\small $^{2}$\IGSV} \\
     {\small $^{3}$\Liverpool} \\
 }
+\newcommand{\PreprintDOI}{10.31223/X58G7C}
```
Comment (santisoler) [inline manuscript/preprint.tex:48]: I would move this to `variables.tex`.

**Review by santisoler — APPROVED** [review]
Thanks @leouieda ! I noticed that I missed the DOI on the preprint. We can update the EarthArXiv preprint after this is merged. I would like to experiment a little bit if we can avoid having the Figure 1 on its own page, though. So let's merge this, experiment with the figure and then update the preprint. Do you agree?

File: manuscript/preprint.tex (hunk 2 above)
Comment (santisoler) [reply manuscript/preprint.tex]: I've just move the DOI to `variables.tex` and also made some small refactors.

Comment (leouieda) [reply manuscript/preprint.tex]: 👍🏾

**Comment by leouieda** [comment]
Agreed! Merging this in now.

**Pull request merged**